dev-infrastructure/modules/aks-cluster-base.bicep

```diff
@@ -404,12 +404,13 @@ resource acrRg 'Microsoft.Resources/resourceGroups@2023-07-01' existing = [
   }
 ]
 
-module acrPullRole 'acr-pull-permission.bicep' = [
+module acrPullRole 'acr-permissions.bicep' = [
   for (_, i) in acrResourceGroups: {
     name: guid(acrRg[i].id, aksCluster.id, acrPullRoleDefinitionId)
     scope: acrRg[i]
     params: {
       principalId: aksCluster.properties.identityProfile.kubeletidentity.objectId
+      grantPushAccess: true
```

Comment:
Should we rename this module since it now can grant push access too?

Reply:
why do we allow it to push to ACR?

Reply:
i think i understand what you want to accomplish. please use https://github.com/Azure/ARO-HCP/pull/398#issuecomment-2251842547 to continue the discussion

Reply:
This push is here by mistake. I added it for testing, purposes. Let me remove it please.